cmd/idracsdkapps/splunkapp/main.go

```diff
@@ -263,13 +280,13 @@ func main() {
 
 	for {
 
-		req, err := http.NewRequest("GET", "https://10.239.37.136/redfish/v1/SSE?$filter=EventFormatType%20eq%20MetricReport", nil)
+		req, err := http.NewRequest("GET", "https://100.69.104.132/redfish/v1/SSE?$filter=EventFormatType%20eq%20MetricReport", nil)
 		if err != nil {
 			fmt.Println("NewRequest GET", req, err)
 		}
 		fmt.Println("NewRequest GET", req, err)
 
-		req.SetBasicAuth("root", "VMware1!")
+		req.SetBasicAuth("root", "Dell1234")
```

Comment:
We should get the authentication piece via dbus.  Jitendra has some great code on this.  I'll dig it up.

Reply:
Same as above.